charts/athens-proxy/README.md

```diff
@@ -1,6 +1,6 @@
 # Athens Proxy Helm Chart: athens-proxy
 
-![Version: 0.14.7](https://img.shields.io/badge/Version-0.14.7-informational?style=flat-square) ![AppVersion: v0.16.0](https://img.shields.io/badge/AppVersion-v0.16.0-informational?style=flat-square)
+![Version: 0.14.8](https://img.shields.io/badge/Version-0.14.8-informational?style=flat-square) ![AppVersion: v0.16.0](https://img.shields.io/badge/AppVersion-v0.16.0-informational?style=flat-square)
```

Comment:
changes to the README must go into the README.md.gotmpl which used by "helm-docs" to generate the README.

Reply:
nvm, I just realised that `helm-docs` obviously does change the readme, so you did everything right 👍